src/lib/stores/storable.ts

```diff
@@ -0,0 +1,35 @@
+import { writable } from 'svelte/store';
+import { BROWSER } from 'esm-env';
```

Comment:
There's also `browser` from `'$app/env'`; I'm curious as to the difference?

Reply:
@JoeCap08055 It is the same value. I was trying to fix import errors that were being thrown in tests despite adding $app to the vite config and vitest config.